develop/dev-guide-sample-application-python-mysqlclient.md

```diff
@@ -1,292 +1,238 @@
 ---
-title: Build a Simple CRUD App with TiDB and mysqlclient
-summary: Learn how to build a simple CRUD application with TiDB and mysqlclient.
+title: Connect TiDB with mysqlclient
+summary: Learn how to connect TiDB using mysqlclient. And you can find sample code snippets that works with TiDB with mysqlclient.
```

Comment:
The second person ("you") gives a conversational tone and speaks directly to the reader, but I am not sure if it should be used in the summary.

with TiDB and mysqlclient?